vite.config.ts

```diff
@@ -24,6 +25,8 @@ export default defineConfig({
   },
 
   plugins: [
+    Preview(),
```

Comment:
Can you add it after `Inspect()` to have all dev plugins together?

Reply:
Sorry for late. 🙏 Preview plugin need to modify SFC code before @vitejs/plugin-vue, otherwise SFC parse will be broken by `<preview>` block, so I think `Preview()` should be put before `Vue()`.